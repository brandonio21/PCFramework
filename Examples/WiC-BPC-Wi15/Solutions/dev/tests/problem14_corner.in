72
13
46
20
5
1
30
1024
13337
0
49
144
