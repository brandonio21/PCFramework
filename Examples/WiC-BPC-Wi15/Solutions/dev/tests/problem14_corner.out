8x9
1x13
2x23
4x5
1x5
1x1
5x6
32x32
1x13337
0x0
7x7
12x12
